dynamo/plot/scatters.py

```diff
@@ -355,7 +421,7 @@ def scatters(
         point_size = 16000.0 / np.sqrt(adata.shape[0]) * pointsize
 
     scatter_kwargs = dict(
-        alpha=0.1,
+        alpha=alpha,
```

Comment:
same as above, set alpha to be 0.1 by default, calpha is set to be 0.4

Reply:
Should be resolved automatically after resolving the previous one?